README.md

```diff
@@ -80,6 +80,32 @@ Since 0.9.1:
 * `xml.validation.disallowDocTypeDecl`: Enable/disable if a fatal error is thrown if the incoming document contains a DOCTYPE declaration. Default is `false`.
 * `xml.validation.resolveExternalEntities`: Enable/disable resolve of external entities. Default is `false`.
         
+## Custom XML Extensions
+
+The [LSP4XML Language Server](https://github.com/angelozerr/lsp4xml) can be extended to support custom completion, hover, validation, rename, etc by using Java SPI feature. vscode-xml gives the capability to start the XML language server with your custom JAR which provides your custom XML support.
```

Comment:
"by using Java SPI feature."

Not a big deal, but it would sound better if it was "by using the Java SPI feature" or "by using Java SPI."

Reply:
@xorye updated